packages/grafana-ui/src/components/TimePicker/TimeRangeInput.tsx

```diff
@@ -45,19 +66,34 @@ export const TimeRangeInput: FC<Props> = ({
     onChange(timeRange);
   };
 
+  const onRangeClear = (event: MouseEvent<HTMLDivElement>) => {
+    event.stopPropagation();
+    const from = '';
+    const to = '';
+    onChange({ from, to, raw: { from, to } });
```

Comment:
I think we should allow `dateTime(null)`. moment  accepts the null values producing date object that is invalid (isValid === false). Ref https://stackoverflow.com/a/57817236

Reply:
Yeah, that seems like a good option. It's just the api often stores time values in a string and returns empty strings when there's no value.

Reply:
Ok, now I see what you mean, indeed seems like `dateTime(null)` is working, so I can get rid of `InputTimeRange` interface. Just need to mention in the docs that `dateTime(null)` should be provided for time range values.

Reply:
perfect!